src/test/ui/consts/static-ice.rs

```diff
@@ -0,0 +1,27 @@
+// check-pass
```

Comment:
Could you pick a more descriptive file name (`extern-static-validity-ice.rs` or so) and/or add a comment? The problem is that we have a static referring to an `extern static`, and validity checking needs the static's size to make sure the reference does not dangle.

Reply:
Can I just refer to the two issues? The sites where the ICE occurred was fairly random.

Reply:
Sure (I thought it was only one of them, sorry).